sklearn/feature_selection/from_model.py

```diff
@@ -130,6 +130,27 @@ class SelectFromModel(MetaEstimatorMixin, SelectorMixin, BaseEstimator):
 
     threshold_ : float
         The threshold value used for feature selection.
+
+    Examples
+    --------
+    >>> from sklearn.feature_selection import SelectFromModel
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> X = [[ 0.86932251, -1.3881055 ,  0.3130677 ],
```

Comment:
It's hard for the reader to see why the result here is sensible when the data i appears random. Use a simpler example perhaps. Or at least cut this back to 2 significant figures rather than 10

Reply:
How about using an easier dataset for an example:

```
    >>> from sklearn.datasets import load_boston
    >>> from sklearn.feature_selection import SelectFromModel
    >>> from sklearn.linear_model import LassoCV

    >>> boston = load_boston()
    >>> X, y = boston['data'], boston['target']
    >>> X.shape
    (506, 10)
    >>> clf = LassoCV(cv=5)
    >>> sfm = SelectFromModel(estimator=clf, max_features=3)
    >>> sfm.fit(X, y)
    >>> sfm.transform(X).shape
    (506, 3)
    >>> boston.feature_names[sfm.get_support()]
    ['RM', 'DIS', 'PTRATIO']
```

Reply:
I have reduced the decimals to 2 and you're right, its much more readable now